collector/prometheus_collector.go

```diff
@@ -17,13 +17,14 @@ package collector
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"math"
+	"io"
 	"net/http"
-	"strconv"
-	"strings"
 	"time"
 
+	dto "github.com/prometheus/client_model/go"
```

Comment:
none-the-less, something more expressive might be in order (perhaps `rawmodel` or `rawtypes`?)